### ISSUE TITLE
Timers warning in syslog

### ISSUE THREAD

**Issue by de-es** [Skycoder42/dolphin-seafile-plugin#6]
Everything is working fine, but there are the following warnings spaming the syslog when the plugin scans/enters Seafile synchronized directories.

```
QObject::killTimer: Timers cannot be stopped from another thread
QObject::startTimer: Timers cannot be started from another thread
```
If more files/directories are contained, more messages will appear.

Is this something that is fixable? Thanks in advance.

**Comment by Skycoder42** [comment]
This seems to be an undetected multithreading issue - have you experienced any other errors beside these warnings?

**Comment by de-es** [comment]
No, these are the only errors. Let me know if I can help to narrow the problem further down.
